.github/workflows/actions.yml

```diff
@@ -127,3 +129,71 @@ jobs:
         grunt test
       env:
         CI: true
+  publish:
+    name: Publish to PyPI registry
+    needs:
+      - build_python
+      - build_javascript
+    runs-on: ubuntu-latest
+
+    env:
+      PY_COLORS: 1
+      TOXENV: packaging
+
+    steps:
+      - name: Switch to using Python 3.6 by default
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install tox
+        run: python -m pip install --user tox
+      - name: Check out src from Git
+        uses: actions/checkout@v2
+        with:
+          # Get shallow Git history (default) for tag creation events
+          # but have a complete clone for any other workflows.
+          # Both options fetch tags but since we're going to remove
+          # one from HEAD in non-create-tag workflows, we need full
+          # history for them.
+          fetch-depth: >-
```

Comment:
Also not clear to me why we are deleting tags here, but I think just setting this to `0` is OK: pytest-html is a tiny repo so cloning the entire history is fine IMHO.